From 145f86d0e98f5525e69a86c15be3f932104b5129 Mon Sep 17 00:00:00 2001
From: Mu Qiao <qiaomuf@gentoo.org>
Date: Thu, 22 Dec 2011 09:57:26 +0800
Subject: [PATCH 4/4] ifnet: remove system prefix

As all connections are system connection so the prefix is meaningless.
Drop it now.
---
 po/POTFILES.in                                 |    1 -
 src/settings/plugins/ifnet/connection_parser.c |   12 ++----------
 2 files changed, 2 insertions(+), 11 deletions(-)

diff --git a/po/POTFILES.in b/po/POTFILES.in
index f021275..20b60d3 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -26,6 +26,5 @@ src/nm-device-olpc-mesh.c
 src/nm-manager.c
 src/nm-netlink-monitor.c
 src/settings/plugins/ifcfg-rh/reader.c
-src/settings/plugins/ifnet/connection_parser.c
 src/settings/nm-settings-utils.c
 
diff --git a/src/settings/plugins/ifnet/connection_parser.c b/src/settings/plugins/ifnet/connection_parser.c
index 66ce318..97bc7ff 100644
--- a/src/settings/plugins/ifnet/connection_parser.c
+++ b/src/settings/plugins/ifnet/connection_parser.c
@@ -45,12 +45,6 @@
 #include "connection_parser.h"
 #include "nm-ifnet-connection.h"
 
-static const char *
-get_prefix (void)
-{
-	return _("System");
-}
-
 static void
 update_connection_id (NMConnection *connection, const char *conn_name)
 {
@@ -62,11 +56,9 @@ update_connection_id (NMConnection *connection, const char *conn_name)
 
 	name_len = strlen (conn_name);
 	if ((name_len > 2) && (g_str_has_prefix (conn_name, "0x"))) {
-		gchar * conn_name_printable = utils_hexstr2bin (conn_name + 2, name_len - 2);
-		idstr = g_strdup_printf ("%s (%s)", get_prefix (), conn_name_printable);
-		g_free (conn_name_printable);
+		idstr = utils_hexstr2bin (conn_name + 2, name_len - 2);
 	} else
-		idstr = g_strdup_printf ("%s (%s)", get_prefix (), conn_name);
+		idstr = g_strdup_printf("%s", conn_name);
 	uuid_base = idstr;
 	uuid = nm_utils_uuid_generate_from_string (uuid_base);
 	setting =
-- 
1.7.3.4

